### DISCUSSION TITLE
Does Leptos reactivity work similar to Mobx?

### DISCUSSION THREAD

**Discussion by ChocolateLoverRaj** [leptos-rs/leptos#2351] [Q&A] [answered]
I've used [Mobx](https://mobx.js.org/README.html) with React before and after trying out Leptos a little it seems like the reactive graph is made in a way similar to how Mobx does it. Are there any big differences?

**Comment by ChocolateLoverRaj** [comment]
I looked at https://book.leptos.dev/appendix_reactive_graph.html#appendix-how-does-the-reactive-system-work and it seems like Leptos is pretty similar to Reactively, which I think is similar to Mobx.

**Comment by gbj** [comment, marked as answer]
In terms of implementation, it is unrelated to MobX. In terms of concepts, there's a pretty 1:1 conceptual mapping:
- observables <> signals
- computeds <> memos
- effects <> reactions
